### DISCUSSION TITLE
save To Stream

### DISCUSSION THREAD

**Discussion by HossienMehdikhah** [SixLabors/ImageSharp#1379] [Q&A] [answered]
### Prerequisites

- [ ] I have written a descriptive issue title
- [ ] I have verified that I am running the latest version of ImageSharp
- [ ] I have verified if the problem exist in both `DEBUG` and mode
- [ ] I have searched [open](https://github.com/SixLabors/ImageSharp/issues) and [closed](https://github.com/SixLabors/ImageSharp/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported

### Description
hi,
I Try to save image to a Memorystream but when copy it to filestream and save it, created file cant be opened,
i think there is some bug for save image to proccess,
i use .net core 3.1. console app

**Comment by JimBobSquarePants** [comment, marked as answer]
Are you resetting the memorystream position to 0 after saving to it?

**Comment by brianpopow** [comment]
@JimBobSquarePants should we convert this to a discussion? I dont think this is a bug.

**Reply by HossienMehdikhah** [reply]
no i think i change my freamwork so i dont sure this problem define as a bug.
so close it issue.please
